### PR TITLE
avifenc: add page

### PR DESCRIPTION
**Pull request by muhlinux**
- [x] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** 0.9.3

### CODE REVIEW

**Comment by navarroaxel** [comment]
@muhlinux Could you please sign the license to pass the CLA check?

File: pages/linux/avifenc.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# avifenc
+
+> AV1 Image File Format (AVIF) encoder.
+> More information: <https://aomediacodec.github.io/av1-avif/>.
+
+- Convert an image from png to avif:
```
Comment (navarroaxel) [inline pages/linux/avifenc.md:6]:
```suggestion
- Convert a specific PNG image to AVIF:
```

File: pages/linux/avifenc.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+`avifenc image.png image.avif`
```
Comment (navarroaxel) [inline pages/linux/avifenc.md:8]:
```suggestion
`avifenc {{path/to/image.png}} {{path/to/image.avif}}`
```

File: pages/linux/avifenc.md [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+
+- Encoder speed (0-10, slowest-fastest, default: 6):
+
+`avifenc --speed 2 image.png image.avif`
```
Comment (navarroaxel) [inline pages/linux/avifenc.md:12]:
```suggestion
`avifenc --speed {{2}} {{path/to/image.png}} {{path/to/image.avif}}`
```

File: pages/linux/avifenc.md [hunk 4]
```diff
@@ -0,0 +1,12 @@
+# avifenc
+
+> AV1 Image File Format (AVIF) encoder.
+> More information: <https://aomediacodec.github.io/av1-avif/>.
+
+- Convert an image from png to avif:
+
+`avifenc image.png image.avif`
+
+- Encoder speed (0-10, slowest-fastest, default: 6):
```
Comment (navarroaxel) [inline pages/linux/avifenc.md:10]: Based on the `zstd` page
```suggestion
- Encode with a specific speed, where 0=slowest, 10=fastest, and 6=default:
```

**Review by navarroaxel — CHANGES_REQUESTED** [review]

**Review by marchersimon — APPROVED** [review]
Looks good after ＠navarroaxel's comments have been addressed.
Thanks

**Review by navarroaxel — APPROVED** [review]

**Comment by navarroaxel** [comment]
@muhlinux could you please sign the license to pass the CLA check?

**Pull request closed without merging**